operator/config/samples/component-integration-installed/moduletemplates/example/blobs/sha256:9d37fb366e4371b44c94496f3db276bd6f3a1019c9d44077b689c68ec6486c1e

```diff
@@ -0,0 +1,7 @@
+configs:
+  - name: some-random-chart-name
+    clientConfig: "CreateNamespace=true,Namespace=jet-cert"
+    overrides: "x=1"
+  - name: some-other-random-chart-name
+    clientConfig: "Namespace=my-amazing-ns"
+    overrides: "x=1"
```

Comment:
we do not need the blobs folder or?

Reply:
blobs are generated by the component-cli for the archive, so nothing from me